qoqo/python_tests/test_circuit.py

```diff
@@ -45,5 +46,17 @@ def test_circuit(measurement):
     assert len(circuit) > 0
 
 
+# def test_struqture_2_0():
+#     """Testing the new 2.0 struqture serialisation."""
+#     hamiltonian = struqture_py.spins.SpinHamiltonianSystem()
+#     hamiltonian.add_operator_product(struqture_py.spins.PauliProduct().z(0).x(2), 1.0)
+#     hamiltonian.add_operator_product("1Z", 2.0)
+
+#     circuit = Circuit()
+#     circuit += ops.ApplyConstantSpinHamiltonian(hamiltonian, 5.0)
+#     circuit += ops.ApplyTimeDependentSpinHamiltonian(hamiltonian, [2.0], {"omega": [1.0]})
+#     assert len(circuit) > 0
+
+
```

Comment:
This is here for future reference I guess?

Reply:
Ah no I have removed this, this test is only meaningful with a manual install of struqture 2.0, which isn't possible in the pipeline